ddtrace/contrib/internal/openai/_endpoint_hooks.py

```diff
@@ -248,7 +249,7 @@ def _record_request(self, pin, integration, instance, span, args, kwargs):
             if kwargs.get("stream_options", {}).get("include_usage", None) is not None:
                 # Only perform token chunk auto-extraction if this option is not explicitly set
                 return
-            span._set_ctx_item("_dd.auto_extract_token_chunk", True)
+            core.set_item("_dd.auto_extract_token_chunk", True)
```

Comment:
We'll want to keep the exact reference to the span, from `core.set_item(span=None)` I don't see it reliably giving us the span we want.